src/mixins/createStatefulChildrenMixin.ts

```diff
@@ -127,9 +127,24 @@ function manageChildren(evt: StateChangeEvent<StatefulChildrenState>): void {
 
 	/* Assume this function cannot be called without the widget being in the management map */
 	const internalState = managementMap.get(parent);
+
 	/* Initialize cache */
-	const { cache = new Map<string, Child>() } = internalState;
+	const { cache = new Map<string | symbol, Child>() } = internalState;
+
 	if (!internalState.cache) {
+		/* register initial children in the internal cache */
+		if (isList(parent.children)) {
+			parent.children.forEach((widget: any) => {
+				const id = internalState.registry.identify(widget);
```

Comment:
I don't believe this code is actually needed, so have removed it and left it initializing with an empty cache of children.